README.md

```diff
@@ -0,0 +1,36 @@
+# Terraform Static Website Infrastructure
+
+This repository presents the use of a module with an infrastructure for hosting a React application with AWS S3 and AWS CloudFront with the possibility of creating domain aliases with AWS Route53 and generating a certificate with Amazon Certificate Manager.

```

Comment:
I feel like the word "presents" here feels a little out of place. Additionally, whether a module was used or not and what AWS services are utilized to provision infrastructure aren't exactly relevant to the first paragraph of the repository's summary. Here's my suggestion, separating which services are used into another paragraph and dropping the mention of a module altogether.

```suggestion
This repository contains Terraform configuration that can be used to provision
cloud infrastructure resources on AWS to host a React application.

The AWS services used are CloudFront and S3. If a domain name is provided,
aliases are created using Route53, and a cerificate is requested from Amazon
Certificate Manager.
```